jsoniter-scala-benchmark/shared/src/main/scala/com/github/plokhotnyuk/jsoniter_scala/benchmark/GitHubActionsAPIReading.scala

```diff
@@ -50,6 +50,9 @@ class GitHubActionsAPIReading extends GitHubActionsAPIBenchmark {
   @Benchmark
   def ninnyJson(): GitHubActionsAPI.Response = Json.parseArray(ArraySeq.unsafeWrapArray(jsonBytes)).to[GitHubActionsAPI.Response].get
 
+  @Benchmark
+  def ninnyJsonUJson(): GitHubActionsAPI.Response = translateUJson(ujson.read(jsonBytes)).to[GitHubActionsAPI.Response].get
```

Comment:
BEWARE: [uJson parser is unsafe](https://github.com/plokhotnyuk/jsoniter-scala/pull/325)

Reply:
Noted
I just have it in there for benchmarking out of curiosity anyway

Reply:
Please withdraw it into a separate branch. I'm not going to merge it until vulnerability will be fixed.